action.yml

```
@@ -47,11 +47,14 @@ runs:
         -X GET https://api.github.com/repos/$GITHUB_REPOSITORY/releases/tags/$TAG \
         -H "Accept: application/vnd.github.v3+json")
         if [[ $RESP_CODE != "200" ]]; then
-          echo "Unable to check tag status for $TAG - HTTP response code was $RESP_CODE"
-          exit 1
+          RESP_MSG=$(jq .message __response.json)
+          if [[ $RESP_MSG != '"Not Found"' ]]; then
```

```suggestion
          RESP_MSG=$(jq -r .message __response.json)
          if [[ $RESP_MSG != "Not Found" ]]; then
```
https://stackoverflow.com/a/44656583

---

@GuySartorelli still need to implement this one

---

Oops, I made the change but for some reason didn't push... sorry about that. Done now.